<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FPGADeviceWidget</class>
 <widget class="QWidget" name="FPGADeviceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QRadioButton" name="select_radioButton">
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="filePath_lineEdit"/>
   </item>
   <item>
    <widget class="QPushButton" name="selectConfigFile_pushButton">
     <property name="text">
      <string>选择配置文件</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="verify_checkBox">
     <property name="text">
      <string>verify</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="parallel_checkBox">
     <property name="text">
      <string>parallel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
